lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
+var Promise = require("bluebird");
+var crypto = require("crypto");
+
+var field_type_hashed_text = new Sealious.ChipTypes.FieldType({
+	name: "hashed-text",
+	extends: "text",
+	get_description: function(context, params){
+		return "Hash text with chosen algorithm (default md5)";
+	},
+	is_proper_value: function(accept, reject, context, params, new_value){
+		var pattern_array = [];
+		if (params instanceof Object){
+			if (params.digit)
+				pattern_array.push(new RegExp("(.*[0-9]+.*){" + params.capitals + "}"));
+			if (params.capitals)
+				pattern_array.push(new RegExp("(.*[A-Z]+.*){" + params.capitals + "}"));
```

Comment:
I propose we change the params names to `n_capitals_required` and `n_digits_required`